src/pyfmi/fmil3_import.pxd

```diff
@@ -237,6 +237,22 @@ cdef extern from 'fmilib.h':
         pass
     ctypedef fmi3_xml_alias_variable_t fmi3_import_alias_variable_t
 
+    cdef struct fmi3_xml_type_definitions_t:
```

Comment:
A few unused until further development, I propose to the remove those unused for now. Also, some names changes here for FMI3, so we can revisit using the correct ones when we actually start using them.